Source/player.cpp

```diff
@@ -319,16 +319,11 @@ void ScrollViewPort(const PlayerStruct &player, _scroll_direction dir)
 	}
 }
 
-bool PlrDirOK(int pnum, Direction dir)
+bool PlrDirOK(PlayerStruct &player, Direction dir)

```

Comment:
```suggestion
bool PlrDirOK(const PlayerStruct &player, Direction dir)
```
This can take a const reference once PosOkPlayer does.